src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+  users: User[];
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit, users }) => {
+  const [title, setTitle] = useState('');
+  const [UserId, setUserId] = useState(0);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserError, setHasUserError] = useState(false);
```

Comment:
You can also create a state for an error message instead of the boolean and check if it's not empty

Reply:
smart